scrapers/errata.py

```diff
@@ -48,12 +48,9 @@ def update_neo4j(self, advisories):
         for advisory in advisories:
             count += 1
             log.info('Processing advisory {0}/{1}'.format(count, len(advisories)))
-            # The content_types column is a string with YAML in it, so convert it to a list
-            content_types = yaml.safe_load(advisory['content_types'])
```

Comment:
Could you remove the `import yaml` line in commit ad5c783353c59b537c32e714f90fca5f30e11c29? Also could you please remove the dependency for pyyaml in `scraper-requirements.txt` and the installation of `python3-PyYAML` in `Dockerfile-scrapers` since this is the only location it was used?